client/state/plans/test/selectors.js

```diff
@@ -51,94 +52,42 @@ describe( 'selectors', () => {
 		} );
 	} );
 
-	describe( '#getPlanPriceObject()', () => {
-		it( 'should return price plan object', () => {
-			const state = getStateInstance();
-			const priceObject = getPlanPriceObject( state, 1003 );
-			expect( priceObject ).to.eql( {
-				currencySymbol: '$',
-				decimalMark: '.',
-				dollars: 99,
-				cents: 0
-			} );
-		} );
-		it( 'should return monthly price plan object', () => {
-			const state = getStateInstance();
-			const priceObject = getPlanPriceObject( state, 1003, true );
-			expect( priceObject ).to.eql( {
-				currencySymbol: '$',
-				decimalMark: '.',
-				dollars: 8,
-				cents: 25
-			} );
-		} );
-		it( 'should return null when plan is not available', () => {
-			const state = getStateInstance();
-			const priceObject = getPlanPriceObject( state, 44, true );
-			expect( priceObject ).to.eql( null );
-		} );
-		it( 'should handle JPY localization correctly', () => {
-			const state = {
+	describe( '#getPlanRawPrice()', () => {
+		it( 'should return annual raw price', () => {
+			const state = deepFreeze( {
 				plans: {
 					items: [ {
-						product_id: 2000,
-						product_name: 'Premium',
-						formatted_price: '¥11,800',
-						raw_price: 11800
-					} ],
-					requesting: false,
-					error: false
+						product_id: 1003,
+						raw_price: 99
+					} ]
 				}
-			};
-			const priceObject = getPlanPriceObject( state, 2000, true );
-			expect( priceObject ).to.eql( {
-				currencySymbol: '¥',
-				decimalMark: '.',
-				dollars: 983,
-				cents: 0
 			} );
+			const price = getPlanRawPrice( state, 1003 );
+			expect( price ).to.eql( 99 );
 		} );
-		it( 'should handle AUD localization correctly', () => {
-			const state = {
+		it( 'should return monthly price plan object', () => {
+			const state = deepFreeze( {
```

Comment:
I'm wondering, why we need the `deepFreeze` here? To mimic the "real" state as much as possible?